tutorials/xr/a_better_xr_start_script.rst

```diff
@@ -0,0 +1,448 @@
+
+This signal is emitted by OpenXR when our game becomes visible but is not focussed. This is a bit of a weird description in OpenXR but it basically means that our game has just started and we're about to switch to the focussed state next, that the user has opened a system menu or the users has just took their headset off.
+
+On receiving this signal we'll update our focussed state, we'll change the process mode of our node to disabled which will pause processing on this node and it's children, and emit our ``focus_lost`` signal.
+
+If you've added this script to your root node, this means your game will automatically pause when required. If you haven't, you can connect a method to the signal that performs additional changes.
+
+.. note::
+
+  While your game is in visible state because the user has opened a system menu, Godot will keep rendering frames and head tracking will remain active so your game will remain visible in the background.
+  However controller and hand tracking will be disabled until the user exits the system menu.
+
+.. tabs::
+  .. code-tab:: gdscript GDScript
+
+    ...
+
+    # Handle OpenXR visible state
+    func _on_openxr_visible_state() -> void:
+        # We always pass this state at startup,
+        # but the second time we get this it means our player took off their headset
+        if xr_is_focussed:
+            print("OpenXR lost focus")
+
+            xr_is_focussed = false
+
+            # pause our game
+            process_mode = Node.PROCESS_MODE_DISABLED
+
+            emit_signal("focus_lost")
+ 
+    ...
+
+  .. code-tab:: csharp
+
+    ...
+
+        /// <summary>
+        /// Handle OpenXR visible state
+        /// </summary>
+        private void OnOpenXRVisibleState()
+        {
+            // We always pass this state at startup,
+            // but the second time we get this it means our player took off their headset
+            if (_xrIsFocused)
+            {
+                GD.Print("OpenXR lost focus");
+
+                _xrIsFocused = false;
+
+                // Pause our game
+                ProcessMode = ProcessModeEnum.Disabled;
+
+                EmitSignal(SignalName.FocusLost);
+            }
+        }
+
+    ...
+
+On focussed state
+-----------------
+
+This signal is emitted by OpenXR when our game gets focus. This is done at the completion of our startup, but it can also be emitted when the user exits a system menu, or put their headset back on.
+
+Note also that when your game starts while the user is not wearing their headset, the game stays in 'visible' state until the user puts their headset on.
+
+.. warning::
+
+  It is thus important to keep your game paused while in visible mode. If you don't the game will keep on running while your user isn't interacting with your game.
+  Also when the game returns to focussed mode, suddenly all controller and hand tracking is reenabled and could have game breaking consequences if you do not react to this accordingly.
```

Comment:
```suggestion
  Also when the game returns to focussed mode, suddenly all controller and hand tracking is re-enabled and could have game breaking consequences if you do not react to this accordingly.
```

I haven't done a full read through of everything yet, but this error is why the build is failing.

Reply:
Cheers, I went to bed straight after submitting so hadn't checked the CI